java/bom/pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>org.apache.arrow</groupId>
   <artifactId>arrow-bom</artifactId>
-  <version>15.0.0</version>
+  <version>siren-15.0.0</version>
```

Comment:
why this change, can you please apply the patch?

Reply:
The version has been fixed: 51d8de67c9bce8ac4e1886244d6f94246d888924